FirebaseAuth/Tests/Sample/ApiTests/FIRAuthApiTestsBase.h

```diff
@@ -18,7 +18,7 @@
 
 #import <FirebaseCore/FIRApp.h>
 #import "AuthCredentials.h"
-#import "FirebaseAuth.h"
+#import <FirebaseAuth/FirebaseAuth.h>
```

Comment:
`Tests/Sample` still seems unformatted, though I suppose that's specifically because of an exclusion in style.sh?

Reply:
Right.